src/Command/AddPriceHistory/CreateApprenticeshipPriceChangeCommandHandler.cs

```diff
@@ -23,7 +23,19 @@ public CreateApprenticeshipPriceChangeCommandHandler(IApprenticeshipRepository a
             CancellationToken cancellationToken = default)
         {
             var apprenticeship = await _apprenticeshipRepository.Get(command.ApprenticeshipKey);
-            apprenticeship.AddPriceHistory(command.TrainingPrice, command.AssessmentPrice, command.TotalPrice, command.EffectiveFromDate, DateTime.Now, PriceChangeRequestStatus.Created, command.UserId, command.Reason);
+            if (string.Equals(command.Requester, PriceChangeRequester.Provider.ToString(), StringComparison.CurrentCultureIgnoreCase))
```

Comment:
No problem with this line, but the file has some unused usings

Reply:
sorted